Styleguide.md

```diff
@@ -39,15 +40,15 @@ function getKey(k) {
 // bad
 const obj = {
   id: 5,
-  name: 'San Francisco',
+  name: "San Francisco",
```

Comment:
How come there are double quotes? Shouldn't single quotes be preferred based on our prettier config?

Reply:
This repo doesn't have prettier config, so whatever are author's default formatter settings in editor/IDE, that will end up happening. By default, it might not even be using prettier as formatter, but VSCode's default formatter.

Easy fix should be to add prettierrc to this repo. @MarkoIvanetic can you please do that?

Reply:
The problem with adding prettier configuration is that it will automatically fix some of the "bad" code.

I think that Marko had prettier installed globally, and this document was formatted with Prettier's default config since there's no prettier specific config in the repo. IMO we should disable prettier for this repository with:
```
// .prettierignore
**
```
instead of adding .prettierrc file

Reply:
Second thought, I added Remark configuration in the other PR, let me know what you think:
https://github.com/infinum/frontend-handbook/pull/204/commits/56c84eda779a91e44dc001ae206f98a000154a17

Reply:
Seems like a good option, I forgot about the fact that prettier would format bad examples as well.

It does however highlight how our handbook has some recommendations about formatting that are probably unnecessary, as prettier will be used on most projects. We could trim the fat a bit by removing formating-only recommendations from handbook

Reply:
disabled my global prettier and had no issues after. remark is a great option 👍